scripts/myMangaDex.js

```diff
@@ -64,6 +64,15 @@ class MyMangaDex {
 		} else if (this.pageUrl.indexOf(urls.history) > -1) {
 			this.historyPage();
 		}
+
+		// clean up existing nodes
```

Comment:
This shouldn't be here, any code related to updating old MyMangaDex modification should be while debugging only, the extension doesn't reload pages on normal use.

Reply:
It is reloaded when it updates but I'll remove that